cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
```

Comment:
@hferentschik I have no problem with refactoring... just not ideal with the timing if you know what I mean ;-)